src/main/java/org/pesmypetcare/webservice/builders/Path.java

```diff
@@ -114,6 +114,21 @@ public static String ofCollection(@NonNull Collections collection, @NonNull Stri
         }
     }
 
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5469160)